### PR TITLE
Disable OpenJ9 XL Specs

### PR DESCRIPTION
**Pull request by AdamBrousseau**
Turn off large heap platforms in the ci builds.
More cleanup can be done related to xl packages
once the dust settles.

Depends #22

Signed-off-by: Adam Brousseau <adam.brousseau@ca.ibm.com>

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM

**Comment by gdams** [comment]
run-tests

**Comment by gdams** [comment]
run tests

**Review by llxia — APPROVED** [review]
LGTM

**Review by M-Davies — CHANGES_REQUESTED** [review]
Can we remove the build config values from the `jdkxx_pipeline_config.groovy` files too?

**Comment by AdamBrousseau** [comment]
@M-Davies is that needed to disable the builds? I was thinking that would be part of the follow up cleanup.

**Comment by M-Davies** [comment]
> @M-Davies is that needed to disable the builds? I was thinking that would be part of the follow up cleanup.

Shouldn't they be removed at the same time? It would allow for an easier revert later down the line in case we decide to add XL specs back in?

**Comment by AdamBrousseau** [comment]
I guess it's up to Adopt. By that argument we should remove all xl related stuff in 1 PR. I don't think there'll be need to revert once the seconds PR cleans up all the xl related code.

**Comment by AdamBrousseau** [comment]
Can I leave it up to Adopt folks to determine if there are other areas that need changes? Website, API, Docker etc. There will be existing released XL builds but going forward there will only be the default build.

**Review by M-Davies — APPROVED** [review]
I'm ok for improving this for now but we should clarify how to remove and/or disable platforms in the future (preferably in the FAQ)

**Review by gdams — APPROVED** [review]

**Comment by M-Davies** [comment]
> I'm ok for improving this for now but we should clarify how to remove and/or disable platforms in the future (preferably in the FAQ)

In fact, we [already clarified it](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/FAQ.md#how-to-enabledisable-a-particular-build-configuration).

**Comment by karianna** [comment]
@AdamBrousseau rebase sorry!

**Comment by AdamBrousseau** [comment]
Rebased. Set as draft until 22 is merged.

**Comment by gdams** [comment]
@AdamBrousseau can we merge this now?

**Comment by AdamBrousseau** [comment]
Yes I think we're ready for this.

**Comment by gdams** [comment]
It would be good to clear up the XL build jobs from the active versions in jenkins. @M-Davies I wonder whether we can modify the generator jobs to cleanup jobs that are no longer defined?

**Pull request merged**
